pydcs_extensions/fa18efg/fa18efg.py

```diff
@@ -228,9 +339,10 @@ class Pylon2:
             2,
             Weapons.GBU_54_V_1_B___LJDAM__500lb_Laser__GPS_Guided_Bomb_LD,
         )
-        AGM_154A___JSOW_CEB__CBU_type_ = (2, Weapons.AGM_154A___JSOW_CEB__CBU_type_)
-        AGM_154C___JSOW_Unitary_BROACH = (2, Weapons.AGM_154C___JSOW_Unitary_BROACH)
```

Comment:
Does the super-hornet no longer support the JSOWs on these pylons? 😅

Reply:
Neither pylon 2 nor pylon 2 support the JSOWs, this is probably because of the pylon limitations.